Gemfile

```diff
@@ -62,7 +62,8 @@ gem 'sinatra'
 
 # Monitoring
 gem 'newrelic_rpm'
-gem 'coveralls', require: false
+gem 'simplecov', require: false
+gem 'simplecov-lcov', require: false
```

Comment:
Is simplecov-lcov really required? The [docs](https://github.com/marketplace/actions/coveralls-github-action) say simplecov is supported. (Trying to test this myself, so far no luck running GitHub Actions locally.)

Reply:
I think [v1](https://github.com/coverallsapp/github-action/tree/release/v1) only supports lcov; support for more formats was added in v2.

Reply:
Okay, let me see if I can update to v2 and get rid of simplecov-lcov.

Reply:
Seems to work: commit a4cf33f66c0a6106fa720c76e07d736f163c3d79, [action run](https://github.com/NZOI/nztrain/actions/runs/7363779985/job/20043556643), [coverage](https://coveralls.io/builds/64806242). Can I add to your branch? (Means one less dependency, less config, and we get nicer local coverage reports.)

Reply:
Yep, feel free.